docs/manual/src/swift/module.md

```diff
@@ -0,0 +1,34 @@
+# Compiling a Swift module
```

Comment:
We do generate a module map, but only while running tests.

Reply:
We do, but I don't think there's a useful way for the consumer to have us generate one for them. Is it worth us trying to expose such a thing for convenience?